### PR TITLE
added example with multiple events

### PR DESCRIPTION
**Pull request by vitaly-zdanevich**

### CODE REVIEW

**Comment by AurelioDeRosa** [comment]
Hi @vitaly-zdanevich.

I'm fine with adding a new example but I'd like to improve the PR a bit. I'll add further comments inline.

File: entries/on.xml [hunk 1]
```diff
@@ -265,13 +265,21 @@ $( "body" ).on( "click", "p", function() {
...
-    <desc>Cancel a link's default action using the <code>.preventDefault()</code> method.</desc>
+    <desc>Cancel a link's default action using the <code>.preventDefault()</code> method:</desc>
     <code><![CDATA[
 $( "body" ).on( "click", "a", function( event ) {
   event.preventDefault();
 });
 ]]></code>
   </example>
+  <example>
+    <desc>Multiple events:</desc>
```
Comment (AurelioDeRosa) [inline entries/on.xml]: Can you expand the sentence to clarify what the ideal goal of your code is?

File: entries/on.xml [hunk 2]
```diff
@@ -265,13 +265,21 @@ $( "body" ).on( "click", "p", function() {
...
+    <code><![CDATA[
+$( "body" ).on( "click mouseleave", function( event ) {
```
Comment (AurelioDeRosa) [inline entries/on.xml]: Can you write a more real-world example so that other people can benefit from it?

**Comment by AurelioDeRosa** [comment]
Hi @vitaly-zdanevich. Is there any specific reason why you have closed this PR?

**Comment by vitaly-zdanevich** [comment]
@AurelioDeRosa I closed this PR just because I created new with correct branch-name and better commit. It would be better to commit to this PR?

**Comment by AurelioDeRosa** [comment]
It's totally fine.

**Pull request closed without merging**